uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/form-sections/leave-type-wizard-form-general.html

```diff
@@ -0,0 +1,52 @@
+<div class="row form-group">
+  <div class="col-md-2">
+    <label>
+      Title
+      <help-text>
+        <p>
+          The Leave Type Title is displayed for everyone on all leave reports and request forms.
+          For example, if you enter "Annual Leave" here, all your staff will see a Leave Type called  "Annual Leave" in their Self Service Portal screens.
+          Use a term that is familiar to them, and a part of your organisation"s leave policies.
+          Other examples: "Annual Leave Full-time" , "Annual Leave Part-time" , "Paid Sick Leave", "Unpaid Sick Leave", "Compensatory Off".
+          <br/><br/>
+          The Leave Type Title is also displayed in the Staff Calendar, but can be made private later in this wizard.
+        </p>
+      </help-text>
+    </label>
+  </div>
+  <div class="col-md-4">
+    <input type="text" class="form-control"/>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <label>What kind of absence type are you looking to create?</label>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-2"
+    ng-repeat="leaveTypeCategory in form.leaveTypeCategories">
+    <div class="well pointer text-center"
+      ng-click="form.leaveTypeCategory = leaveTypeCategory.value">
```

Comment:
`<label>` will require additional CSS because it will not render 100% width of the container. I was thinking about this and think it would make sense to optimise in the next story where we have multiple switchers.